source/documentation/dns/register_defensive_domain.html.md.erb

```diff
@@ -0,0 +1,79 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: Register New Defensive Domain
+last_reviewed_on: 2024-10-16
+review_in: 6 months
+---
+
+# Register New Defensive Domain
+
+This document explains the process of registering a defensive domain at the Ministry of Justice.
+
+The [Defensive Domain Registrations policy](https://security-guidance.service.justice.gov.uk/defensive-domain-registration/#defensive-domain-registrations) is maintained by the Security team. Operations Engineering is not involved in deciding which domains require defensive registration or determining the criteria for such registrations. Our role is to implement the policy once a justified requirement is presented.
+
+## Pre-requisites
+
+Before proceeding with the registration, ensure you have:
+
+- A list of domains requiring defensive registration
+- A copy of the approval email from the Security Team
+- Access to MoJDSD AWS Route53
+
+## Register domain
+
+Follow [AWS Route53 documenation to regsiter domain via the console](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/domain-register.html).
+
+If we are unable to register a domain as it is already owned by someone else, report that back to Requester and Security.
+
+## Create hostedzones in DNS repository
+
+Registering a domain will automatically create a Hostedzone in AWS Route53. These steps are for adding the Hostedzone to management by [DNS](https://github.com/ministryofjustice/dns), along with the standard configuration of DNS records that all defensive domains must have.
+
+-  Add new Hostedzone `.yaml` file to `hostedzones` as per the usual process.
+
+- Add the following configuration which will add all the standard defensive domain records:
+
+```
+---
+'':
+  - ttl: 300
+    type: CAA
+    values:
+      - flags: 0
+        tag: iodef
+        value: mailto:certificates@digital.justice.gov.uk
+      - flags: 0
+        tag: issue
+        value: ;
+  - ttl: 300
+    type: MX
+    value:
+      exchange: .
+      preference: 0
+  - ttl: 172800
+    type: NS
+    values:
+      - ns-xxxx.awsdns-xx.org.
+      - ns-xxxx.awsdns-xx.co.uk.
+      - ns-xxx.awsdns-xx.com.
+      - ns-xxx.awsdns-xx.net.
+  - ttl: 300
+    type: TXT
+    value: v=spf1 -all
+'*._domainkey':
+  ttl: 300
+  type: TXT
+  value: v=DKIM1\; p=
+_dmarc:
+  ttl: 300
+  type: TXT
+  value: v=DMARC1\;p=reject\;sp=reject\;rua=mailto:dmarc-rua@dmarc.service.gov.uk\;
+```
+
+**Note - You must update NS records in the example with actual NS records from Route53 that were created on registration.**
+
+- Raise PR for changes as per the normal process.
+
+- Repeat process for all domains registered.
+
+- Notify Requester and Security when completed.
```

Comment:
Argh, the suggest function is no good for this one. I cleaned it up a little and wrapped the codeblock in yaml. I think this reads better.
```markdown
Sure, here’s the complete version wrapped in a markdown code block:

```markdown
## Add Hosted Zones to the DNS Repository

When a domain is registered, a Hosted Zone is automatically created in AWS Route 53. The steps below outline how to add this Hosted Zone for management in the [DNS repository](https://github.com/ministryofjustice/dns), along with configuring the standard DNS records required for all defensive domains.

1. **Add the Hosted Zone YAML File**  
   Create a new `.yaml` file in the `hostedzones` directory following the standard process.

2. **Configure Standard DNS Records**  
   Add the following configuration to include the standard defensive domain records:

   ```yaml
   ---
   '':
     - ttl: 300
       type: CAA
       values:
         - flags: 0
           tag: iodef
           value: mailto:certificates@digital.justice.gov.uk
         - flags: 0
           tag: issue
           value: ;
     - ttl: 300
       type: MX
       value:
         exchange: .
         preference: 0
     - ttl: 172800
       type: NS
       values:
         - ns-xxxx.awsdns-xx.org.
         - ns-xxxx.awsdns-xx.co.uk.
         - ns-xxx.awsdns-xx.com.
         - ns-xxx.awsdns-xx.net.
     - ttl: 300
       type: TXT
       value: v=spf1 -all
   '*._domainkey':
     ttl: 300
     type: TXT
     value: v=DKIM1\; p=
   _dmarc:
     ttl: 300
     type: TXT
     value: v=DMARC1\;p=reject\;sp=reject\;rua=mailto:dmarc-rua@dmarc.service.gov.uk\;
   /```

   **Note**: Update the `NS` records with the actual values generated by Route 53 during the domain registration process.

3. **Submit a Pull Request**  
   Raise a pull request for the changes following the usual process.

4. **Repeat the Process for All Registered Domains**  
   Complete the above steps for each domain that has been registered.

5. **Notify the Requester and Security Team**  
   Inform both the requester and the Security team once the process is completed for all domains.
```

Reply:
You'll have to remove the " /" just before the closing yaml codeblock.